libnavui-util/src/test/java/com/mapbox/navigation/ui/utils/internal/lifecycle/ViewLifecycleRegistryTest.kt

```diff
@@ -16,16 +17,21 @@ import io.mockk.slot
 import io.mockk.unmockkStatic
 import io.mockk.verify
 import io.mockk.verifyOrder
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import org.junit.After
 import org.junit.Assert
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 
+@OptIn(ExperimentalCoroutinesApi::class)
 class ViewLifecycleRegistryTest {
     @get:Rule
     val instantTaskExecutorRule = InstantTaskExecutorRule()
 
+    @get:Rule
+    val coroutineRule = MainCoroutineRule()
```

Comment:
Had to add the `MainCoroutineRule` here in order to avoid `Module with the Main dispatcher had failed to initialize` errors.